styleguide/gulpfile.js

```diff
@@ -1,12 +1,8 @@
 // npm requirements
 var gulp        = require('gulp'),
-  bump        = require('gulp-bump'),
```

Comment:
@designerneil for each of these removals, can you explain why they aren't needed?

Reply:
bump not used.